src/app/components/BibPage/BibPage.jsx

```diff
@@ -107,17 +107,21 @@ class ItemPage extends React.Component {
       { label: 'Number available', field: 'numAvailable' },
       { label: 'Number of items', field: 'numItems' },
       { label: 'Shelf Mark', field: 'shelfMark' },
-    ]
+    ];
+    let shortenItems = true;
+    if (this.props.location.pathname.indexOf('all') === -1) {
```

Comment:
No. When it doesn't have `all`, it will only show 20 items and the more link. When it has `all` it will display all items.